app/mailers/report_mailer.rb

```diff
@@ -7,6 +7,10 @@ def report_content
     @reported_content = params.fetch(:reported_content)
     study_group = @reported_content.submission.study_group
     exercise = @reported_content.exercise
+
+    # NOTE: This implementation assumes the curse URL is static and does not vary per user.
+    # This is currently valid for all known use cases. However, in dynamic environments—such as
+    # advanced content trees in openHPI or A/B/n testing—this assumption may break.
```

Comment:
Luckily, we don't need to curse the course ;)

```suggestion
    # NOTE: This implementation assumes the course URL is static and does not vary per user.
    # This is currently valid for a majority of use cases. However, in dynamic scenarios (such as
    # content trees in openHPI used in conjunction with A/B/n testing) this assumption may no longer hold true.
```